client/src/components/NationwideMap.vue

```diff
@@ -78,7 +78,14 @@ export default defineComponent({
   },
   methods: {
     zoomToLongLat() {
-      if (this.geoState?.geoids?.lat != null && this.geoState?.geoids.long != null) {
+      if(this.geoState?.geoids?.pwsId?.bounding_box){
+        const {minLat, minLon, maxLat, maxLon} = this.geoState?.geoids?.pwsId?.bounding_box;
+        this.map?.fitBounds([
+          [minLat, minLon],
+          [maxLat, maxLon]
+        ]);
+      }
+      else if (this.geoState?.geoids?.lat != null && this.geoState?.geoids.long != null) {
```

Comment:
I assumed it was a good idea to leave the previous `flyTo` behavior around as a fallback.

Reply:
Agree, thank you!